pytorch_lightning/loggers/tensorboard.py

```diff
@@ -173,3 +175,15 @@ def _get_next_version(self):
             return 0
 
         return max(existing_versions) + 1
+
+    def _sanitize_params(self, params):
```

Comment:
```suggestion
    @staticmethod
    def _sanitize_params(self, params):
```

Reply:
maybe move to base class so other loggers could use it if needed

Reply:
I'll move it to the base class.
The ```_convert_params``` method is not static right now, should I make it static as well, for uniformity?

Reply:
yes, static would be better since there is no use of the class attributes
also thinking about a name like `_serilizable_params` or not, maybe I am overthinking it too much lol

Reply:
I think I'll probably leave it sanitize params, I was thinking about serialize but since there is separate serialization in the checkpoint and this is mainly just for converting for the logger I thought sanitize was better.